src/applications/vaos/services/appointment/index.js

```diff
@@ -841,7 +841,6 @@ export function getCalendarData({ appointment, facility }) {
           'You can join this meeting up to 30 minutes before the start time.',
         location: 'VA Video Connect at home',
         additionalText: [signinText],
-        phone: getFacilityPhone(facility),
```

Comment:
Including a phone number in these two cases was putting a VA facility phone number in the middle of a video or ATLAS appointment, which didn't make sense. Need to verify this against the ticket requirements, though.

Reply:
could this change be causing this message in the logs?
![image](https://user-images.githubusercontent.com/12739849/122797807-de7eb680-d28d-11eb-8254-1db3feb140c3.png)

Reply:
No, that message doesn't come from VAOS tests, as far as I can tell (i.e. I don't see it when just running VAOS tests). If you run all the tests locally with `--reporter list`, you should be able to see the tests that are running when that message is generated.

Reply:
It is actually coming from the VAOS tests. I can see them when I run `yarn test:unit --app-folder vaos --log-level trace` (that's the log-level we use when running the tests on Jenkins). I'm also seeing a lot of other react errors and `Error: Uncaught [TypeError: Cannot read property 'trim' of undefined]` errors.

Reply:
These errors may pre-date your PR, though. I'm looking into that now.

Reply:
Unfortunately they don't appear to. These are [the test logs](http://jenkins.vfs.va.gov/blue/rest/organizations/jenkins/pipelines/testing/pipelines/vets-website/branches/master/runs/11612/nodes/64/steps/99/log/?start=0) from the latest `master` run and [these are the test logs](http://jenkins.vfs.va.gov/blue/rest/organizations/jenkins/pipelines/testing/pipelines/vets-website/branches/vaos-404-cleanup-jsb/runs/1/nodes/64/steps/100/log/?start=0) from your PR run.

Reply:
Oh, I wasn't using the trace level. I didn't know that you all had changed that for the Jenkins build (it used to be debug). There's a ton of output in there that is going to be very difficult to clean up across all the vets-website tests.

There are plenty of React and other errors due to us testing error paths. I've looked through those before and they're generally not accidental. Those trim errors are new, though, and I'm looking into them. I think they're coming from the Breadcrumbs component in the design system, because we only use `trim` in a couple places, but I'm not sure why yet.

Reply:
Ah, I looked into that and I see what's going on. The `log-level trace` is only being used for PRs (and it is a recent change), so that's why I wasn't seeing all those errors in the `master` run (and why I hadn't seen them before). No worries, then. 👍

Reply:
Gotcha. Well, I did look into them, and we have some default mocks that have bad or missing phone numbers. Those fields are required in the data we get from the backend, so we don't handle the case where they're missing in code. And when actually test that the data shows up, we set it to specific numbers.

I think I'll update those default mocks in a separate PR, since they will require some test updates, too.

Reply:
Oh that would be awesome. Thank you for looking into it!